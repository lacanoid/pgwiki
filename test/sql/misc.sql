\pset null _null_

SET client_min_messages = warning;
SET ROLE postgres;

select wiki.page_set('Main_Page','Hello! :)');

select page_title,
       page_body,
       page_regtype,
       page_vector
  from wiki.page;

select rev_comment,rev_length
  from wiki.revision;


