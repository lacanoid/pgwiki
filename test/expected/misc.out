\pset null _null_
SET client_min_messages = warning;
SET ROLE postgres;
select wiki.page_set('Main_Page','Hello! :)');
 page_set 
----------
       15
(1 row)

select wiki.page_set('rss','<rss></rss>'::xml);
 page_set 
----------
       16
(1 row)

select page_title,
       page_body,
       page_regtype,
       page_vector
  from wiki.page;
 page_title |  page_body  | page_regtype | page_vector 
------------+-------------+--------------+-------------
 Main_Page  | Hello! :)   | _null_       | 'hello':1
 rss        | <rss></rss> | xml          | 
(2 rows)

select rev_comment,rev_length
  from wiki.revision;
 rev_comment | rev_length 
-------------+------------
 _null_      |          9
 _null_      |         11
(2 rows)

